apps/contrib/templatetags/contrib_tags.py

```diff
@@ -1,18 +1,28 @@
-import requests
+import os
+
 from django import template
 from django.conf import settings
+from django.core import management
+from django.core.cache import cache
 from django.utils.html import mark_safe
 
 register = template.Library()
 
 
 @register.simple_tag
 def get_external_footer():
+    # The BO footer needs to be included here as we can't include external
+    # html in the browser. Add a cron job to update the file periodically via
+    # the get_footer management command.
     if hasattr(settings, 'BERLIN_FOOTER_URL') and settings.BERLIN_FOOTER_URL:
-        try:
-            response = requests.get(settings.BERLIN_FOOTER_URL, timeout=2.0)
-        except (requests.ConnectTimeout, requests.HTTPError, requests.Timeout):
-            return ''
-        return mark_safe(response.text)
-    else:
-        return ''
+        footer = cache.get('footer')
+        if footer:
+            return footer
+        filepath = settings.MEDIA_ROOT + "/landesfooter.inc"
+        if not os.path.isfile(filepath):
+            management.call_command('get_footer')
```

Comment:
But what happens if it timeouts here? Maybe we could write '' to the filepath then, would that work?

Reply:
I added error handling so it returns '' if the management command failed

Reply:
Cool, thanks!